bundle/src/projects/common/modules/spacefinder.ts

```diff
@@ -81,7 +83,7 @@ const LOADING_TIMEOUT = 5_000;
 const defaultOptions: SpacefinderOptions = {
 	waitForImages: true,
 	waitForInteractives: false,
-	debug: false,
+	pass: 'inline1',
```

Comment:
This is questionable, `tsc` made me do it. There is no appropriate default value. But luckily this property is non-optional so the default will never actually be used.